intra/dialers/split_and_desync.go

```diff
@@ -98,7 +98,15 @@ func DialWithSplitAndDesyncTraceroute(d *protect.RDial, ipp netip.AddrPort, maxT
 	udpAddr.Port = 1 // unset port
 
 	isIPv6 := ipp.Addr().Is6()
-	udpConn, err := d.AnnounceUDP("udp", ":0")
+
+	var networkStr string
```

Comment:
For future refactorers (ie, me) a comment here on why one must avoid dialing happy-eyeballs / dual-socket (`"udp"`) here will help...

Reply:
🥲I feel it isn't worth the effort.